docs/framework/guides/tutorials/implementing-an-inline-widget.md

```diff
@@ -544,11 +544,11 @@ export default class PlaceholderEditing extends Plugin {
 	}
 
 	_defineSchema() {
-		// ...
```

Comment:
We should keep those ellipses below the comment, as shown here: https://github.com/cksource/ckeditor5-internal/issues/2307.